tests/console/sudo.pm

```diff
@@ -47,6 +47,16 @@ sub run {
     my $test_password = 'Sud0_t3st';
     select_console 'root-console';
     zypper_call 'in sudo expect';
+    select_console 'user-console';
+    # Verify that sudo works with the default configuration.
+    if (is_public_cloud) {
+        # No defaults targetpw (public cloud) -> asks for user PW
+        assert_script_run("expect -c 'spawn sudo id -un;expect \"password for $testapi::username\" {send \"$testapi::password\\r\";interact} default {exit 1}' | grep ^root");
+    } else {
```

Comment:
If I got it right... isn't this the actual condition that will land on TW eventually? cc @DimStar77

Reply:
Eventually maybe, but not now and not outside of TW.

Reply:
> If I got it right... isn't this the actual condition that will land on TW eventually? cc @DimStar77

Indeed, eventually, this might be the case. But for now, we want to protect our users from this implementation that is half-baked and definitively not ready